pylint/checkers/base.py

```diff
@@ -261,7 +261,7 @@ def _redefines_import(node):
 
 
 def in_loop(node: nodes.NodeNG) -> bool:
-    """Return whether the node is inside a kind of for loop"""
+    """Return whether the node is inside a kind of for loop."""
```

Comment:
Could you remove these changes introduced by `pydocstringformatter`? You have probably installed the latest `main` commit of the tool which is not currently being used by `pylint`.

Reply:
yes I did, and okay I will